### ISSUE TITLE
Disable dragging up in offline mode

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#2679]
![image](https://user-images.githubusercontent.com/1689815/58473403-c8eb4100-8148-11e9-8f93-5a4153022e57.png)

**Comment by vanshikaarora** [comment]
Hey @teolemon I would like to work on this issue. Could you please translate the code to English. I couldn't get which Activity is this

**Comment by teolemon** [comment]
ContinuousScanActivity, I think. 
The best way to approach it is to try scanning a book (both online and offline) and try dragging up.

**Comment by vanshikaarora** [comment]
> The best way to approach it is to try scanning a book (both online and offline) and try dragging up.

Yeah thanks for this I tried it and got that it's empty in offline mode. I can work on this issue can you please assign this to me :)

**Comment by vanshikaarora** [comment]
Hey @teolemon @RajaVamsi11  I have raised a PR here #2685. Kindly review :)
